.clang-tidy

```diff
@@ -1,5 +1,5 @@
 ---
-Checks:          'clang-analyzer-*,modernize-use-override,readability-inconsistent-declaration-parameter-name'
+Checks:          'clang-analyzer-*, performance-*,readability-*,-readability-identifier-naming,-readability-container-size-empty,-readability-redundant-declaration,modernize-*,-modernize-avoid-bind,-modernize-loop-convert,-modernize-use-using,llvm-namespace-comment,cppcoreguidelines-pro-*,-cppcoreguidelines-pro-type-reinterpret-cast,-cppcoreguidelines-pro-type-vararg,-cppcoreguidelines-pro-bounds-pointer-arithmetic,-cppcoreguidelines-pro-type-const-cast,-cppcoreguidelines-pro-bounds-array-to-pointer-decay,-cppcoreguidelines-pro-bounds-constant-array-index,cert-*,-cert-err58-cpp,-cert-env33-c,-cert-err09-cpp,-cert-err60-cpp,-cert-err61-cpp, -cert-err61-cpp, bugprone-*, google-*,-google-runtime-references,-google-runtime-member-string-references, -google-explicit-constructor, misc-*, -misc-unused-parameters, -misc-macro-parentheses, -misc-throw-by-value-catch-by-reference, -misc-suspicious-enum-usage'
 WarningsAsErrors: ''
 AnalyzeTemporaryDtors: false
 User:            lbonn
```

Comment:
I wonder if this `User: lbonn` line is the source of the `TODO(lbonn)` problems. Is it necessary or wise to keep that line?

Reply:
Yes You are right, but with removed `User: lbonn` it suggest to add everywhere my username, so it doesn't use git blame. I will remove this check